libs/ledger-live-common/src/apps/hw.ts

```diff
@@ -228,26 +245,7 @@ export const listApps = (
           )
           .catch((e) => {
             log("hw", "failed to HSM list apps " + String(e) + "\n" + e.stack);
-
-            if (getEnv("EXPERIMENTAL_FALLBACK_APDU_LISTAPPS")) {
```

Comment:
is there an active reason to drop this? if yes, should you also drop the env related to it as well as the hwListApps implementation? not sure if it would therefore used at all?

Reply:
I'm dropping it because it was introduced 4 years ago and it has never been used or exposed to the UI since.
We maintain the command in case we want to introduce it as part of our flow and **[it's now documented](https://ledgerhq.atlassian.net/wiki/spaces/~5be5d09253edba78d2964720/pages/3753377984/An+attempt+at+APDU+specs)**  on our confluence. I see no reason to maintain that complexity in the hw.ts file in the meantime.

Reply:
then i suggest to drop the unused env 👍

Reply:
sure, you're right. Just did